### PR TITLE
added dutch national flag algo

### LINKED ISSUE #481: [Feature Request] Dutch National Flag algo
### Is your feature request related to a problem? Please describe.
i want to write dutch national flag algorithm under arrays section.assign this issue to me pls.

**Would you like to work on this feature?**  
- [yes ] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by Abhishek2634**
### Description
i added dutch naitonal flag algorithm in algorithm section.

Fixes #481

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [yes ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/algorithms/DutchNationalFlag-algorithm/DutchNationalFlag.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    sol.sortZeroOneTwo(nums);
+
+    cout << "After sorting: ";
+    for (int i = 0; i < nums.size(); i++) {
+        cout << nums[i] << " ";
+    }
+    cout << endl;
+
+    return 0;
+}
```
Comment (ajay-dhangar) [inline docs/algorithms/DutchNationalFlag-algorithm/DutchNationalFlag.md:97]: please close your code example with ```` ``` ````

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

File: docs/algorithms/DutchNationalFlag-algorithm/DutchNationalFlag.md (hunk 1 above)
Comment (Abhishek2634) [reply docs/algorithms/DutchNationalFlag-algorithm/DutchNationalFlag.md:97]: @ajay-dhangar  its done bro!

**Review by ajay-dhangar — APPROVED** [review]
👍

**Pull request merged**